packages/modelviewer.dev/examples/examples-and-docs.html

```diff
@@ -0,0 +1,216 @@
+  <link type="text/css" href="../styles/examples.css" rel="stylesheet" />
+  <link rel="shortcut icon" type="image/png" href="../assets/favicon.png" />
+  <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
+
+  <!-- The following libraries and polyfills are recommended to maximize browser support -->
+
+  <!-- 🚨 REQUIRED: Web Components polyfill to support Edge and Firefox < 63 -->
+  <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+
+  <!-- 💁 OPTIONAL: Intersection Observer polyfill for better performance in Safari and IE11 -->
+  <script src="../node_modules/intersection-observer/intersection-observer.js"></script>
+
+  <!-- 💁 OPTIONAL: Resize Observer polyfill improves resize behavior in non-Chrome browsers -->
+  <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
+
+  <!-- 💁 OPTIONAL: The :focus-visible polyfill removes the focus ring for some input types -->
+  <script src="../node_modules/focus-visible/dist/focus-visible.js" defer></script>
+
+  <style>
+    html {
+      scroll-behavior: smooth;
+    }
+
+    .sidebar {
+      margin: 0;
+      padding: 0;
+      width: 300px;
+      background-color: #f6f6f6;
+      position: fixed;
+      height: 100%;
+      overflow: auto;
+      border: 1px solid #BDBDBD;
+    }
+
+    .outer-content {
+      margin-left: 300px;
+    }
+
+    .inner-content {
+      padding: 10px 40px;
+    }
+
+    p {
+      display: inline;
+    }
+
+    .attribute-name {
+      padding: 10px 20px 10px 20px;
+      margin: 10px 0 20px 0;
+      background-color: #ececec;
+    }
+
+    .attribute-definition {
+      margin: 0 0 20px 0;
+    }
+
+    .category-container {
+      padding-bottom: 20px;
+    }
+
+    .attribute-container {
+      padding-bottom: 20px;
+    }
+
+    .value-table {
+      margin: 0 0 30px 0;
+      width: 50%;
+    }
+
+    table,
+    td,
+    th {
+      border: 1px solid #ececec;
+      border-collapse: collapse;
+    }
+
+    th {
+      font-weight: bold;
+    }
+
+    td:hover {
+      background-color: #ececec;
+    }
+
+    ul.links {
+      margin-top: 5px;
+    }
+
+    hr.solid {
+      border-top: 3px solid #f6f6f6;
+    }
+
+    .header {
+      background-color: #f6f6f6;
+      padding: 10px 20px 10px 40px;
+      width: 100%;
+      position: sticky;
+      top: 0;
+      border-top: 1px solid #BDBDBD;
+      border-bottom: 1px solid #BDBDBD;
+    }
+  </style>
+</head>
+
+<body>
+  <div class="sidebar">
+    <ol>
+      <li><a href="#introduction">Introduction</a></li>
+      <li><a href="#loading">Loading</a></li>
+      <li><a href="#animation">Animation</a></li>
+      <li><a href="#lighting-and-environment">Lighting &amp; Environment</a></li>
+      <li><a href="#model-formats">Model Formats</a></li>
+      <li><a href="#staging-and-camera-controls">Staging &amp; Camera Controls</a></li>
```

Comment:
Didn't you decide on some new categories like Interaction Prompt?

Reply:
we'll have to figure out how they'd work with respect to flipping between Documentation/Examples